region/crateria/central.json

```diff
@@ -3660,12 +3660,123 @@
                   ]
                 }
               ]
+            },
+            {
+              "id": 5,
+              "strats": [
+                {
+                  "name": "G-Mode Morph and Bombs to overload Speed Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Bombs",
+                    {"or": [
+                      {"and": [
+                        "SpringBall",
+                        {"spikeHits": 3},
+                        {"or": [
+                          "h_canArtificialMorphBombHorizontally",
+                          {"spikeHits": 1}
+                        ]}
+                      ]},
+                      {"and": [
+                        "Morph",
+                        {"or": [
+                          "SpaceJump",
+                          {"and": [
+                            {"spikeHits": 1},
+                            {"or": [
+                              "canIframeSpikeJump",
+                              {"spikeHits": 1}
+                            ]}
+                          ]}
+                        ]}
+                      ]},
+                      {"and": [
+                        "h_canArtificialMorphIBJ",
+                        {"spikeHits": 10},
+                        {"or": [
+                          "h_canArtificialMorphBombHorizontally",
+                          {"spikeHits": 2}
+                        ]}
+                      ]},
+                      "h_canArtificialMorphCeilingBombJump"
+                    ]}
+                  ],
+                  "devNote": [
+                    "The IBJ strat has 2 extra spike hits added as a leniency.",
+                    "FIXME Add strat for going down without bombs, by using the uncollected item to overload PLMs."
+                  ]
+                },
+                {
+                  "name": "G-Mode and Grapple to overload Grapple Blocks",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": false
+                    }},
+                    "Morph",
+                    "Grapple",
+                    {"or": [
+                      "canCarefulJump",
+                      {"spikeHits": 1}
```

Comment:
I think this represents being pulled towards the grapple blocks, which should be in a note